docker-for-mac/mutagen-caching.md

```diff
@@ -67,11 +69,58 @@ on the webserver.
 As you edit code on the host, the changes are detected and transferred to the
 container for testing. Changes inside the container (for example, the creation of build artifacts) are detected and transferred back to the host.
 
-## Avoiding synchronizing a subdirectory
+## Adding synchronized directories through the CLI
 
-Although two-way file sync is suitable for many types of files, sometimes containers can generate lots of data which doesn't require copying to the host, for example, debug logs.
+As of Docker Desktop Edge 2.3.2.0 the `:delegated` flag on a shared volume will 
+automatically enable synchronization. This is particularly useful if you want to share
+the synchronization setting with other developers in your team, through your source code.
 
-If your project has a subdirectory that doesn't need to be continuously copied back to the host, then use a named docker volume to bypass the sync.
+## Scripting
+
+Many projects have scripts which modify files in a container and then read them back on
+the host, or the other way around. To support these, Docker Desktop automatically flushes
+file changes around `docker run` calls. Therefore, the following will work as expected:
+
+```
```

Comment:
nit: looks like we're missing some code-hints (`bash` or `console`) here (and some lines below)

Reply:
Fixed